cmd/createcluster_internal_test.go

```diff
@@ -170,19 +226,53 @@ func TestValidNetwork(t *testing.T) {
 		WithdrawalAddr: "0x0000000000000000000000000000000000000000",
 		Network:        "gnosis",
 	}
-	err := validateClusterConfig(ctx, conf)
+	err := validateClusterConfig(ctx, conf.InsecureKeys, conf.NumNodes, conf.Name, conf.WithdrawalAddr, conf.Network)
 	require.Error(t, err, "zero address")
 
 	conf.Network = "goerli"
-	err = validateClusterConfig(ctx, conf)
+	err = validateClusterConfig(ctx, conf.InsecureKeys, conf.NumNodes, conf.Name, conf.WithdrawalAddr, conf.Network)
 	require.NoError(t, err)
 
 	conf.InsecureKeys = true
 
-	err = validateClusterConfig(ctx, conf)
+	err = validateClusterConfig(ctx, conf.InsecureKeys, conf.NumNodes, conf.Name, conf.WithdrawalAddr, conf.Network)
 	require.NoError(t, err)
 
 	conf.Network = "mainnet"
-	err = validateClusterConfig(ctx, conf)
+	err = validateClusterConfig(ctx, conf.InsecureKeys, conf.NumNodes, conf.Name, conf.WithdrawalAddr, conf.Network)
 	require.Error(t, err, "zero address")
 }
+
+// newDefinition returns a new definition with creator field populated.
+func newDefinition(t *testing.T, clusterName string) cluster.Definition {
```

Comment:
instead of exporting logic for a single test, rather just use the `charon/cluster/examples/cluster-definition-002.json` file as a test fixture directly.

Reply:
`charon/cluster/examples/cluster-definition-002.json` file is not a valid definition file as it only has 2 operators and will fail in `validateConfig`

Reply:
done